statsmodels/survey/summary_stats.py

```diff
@@ -0,0 +1,185 @@
+
+    def _stat(self, SurveyDesign,c=None):
+        """                                                                                                                                              
+        Calculate a statistic with possible cluster deletion.                                                                                            
+                                                                                                                                                         
+        Parameters                                                                                                                                       
+        ----------                                                                                                                                       
+        c : int or None                                                                                                                                  
+            If an integer, return the statistic calculated with                                                                                          
+            cluster c deleted.  If c is None, return the statistic                                                                                       
+            calculated for the whole data set.                                                                                                           
+                                                                                                                                                         
+        Returns                                                                                                                                          
+        -------                                                                                                                                          
+        An array containing the statistic calculated on the columns                                                                                      
+        of the dataset.                                                                                                                                  
+        """
+
+        if c is None:
+            return np.dot(SurveyDesign.weights, self.data)
+
+        s = SurveyDesign.sfclust[c]
+        nh = SurveyDesign.ncs[s]
+        self.w = SurveyDesign.weights.copy()
+        self.w[SurveyDesign.strat == s] *= nh / float(nh - 1)
+        self.w[SurveyDesign.sclust == c] = 0
+        return np.dot(self.w, self.data)
+
+class SurveyPercentile(SurveyDesign):
+
+    def __init__(self, SurveyDesign, data, percentile):
+        self.data = data
+        self.cumsum_weights = np.cumsum(SurveyDesign.weights)
+        self.est = [[self._stat(perc, index) for index in range(self.data.shape[1])] for perc in percentile]
+
+    def _stat(self, percentile, col_index):
+
+        percentile *= (self.cumsum_weights[-1] / 100)
```

Comment:
@kshedden I can't find your original comment but I recall you mentioning how I shouldn't pass in quantile as a parameter in _stat, but just make it an attribute as well as column index as a paramter. I think this might raise an issue when trying to calculate se via bootstrap and jackknife because none of the other stats require passing in a column index so jk and boot would have to know I'm working w/ SurveyQuantile.. A similar problem might occur with surveyRatio